tests/helper/helper_oc.go

```diff
@@ -0,0 +1,269 @@
+package helper
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+type OcRunner struct {
+	// path to oc binary
+	path string
+}
+
+// New initializes new OcRunner
+func NewOcRunner(ocPath string) OcRunner {
+	return OcRunner{
+		path: ocPath,
+	}
+}
+
+// Run oc with given arguments
+func (oc *OcRunner) Run(cmd string) *gexec.Session {
+	session := CmdRunner(cmd)
+	Eventually(session).Should(gexec.Exit(0))
+	return session
+}
+
+// CreateRandProject create new project with random name (10 letters)
+// without writing to the config file (without switching project)
+func (oc *OcRunner) CreateRandProject() string {
+	projectName := "project-" + RandString(8)
```

Comment:
why is `project-` prefix needed?

Reply:
Just to make it more specific. No other idea behind this

Reply:
If there is no idea behind it than I wouldn't complicate it like this ;-)

Reply:
Reorder happens based on project names in actual output

Reply:
Ahh...Now i remember, it was a mismatch in the json if reordering happens. To fix the ordering with existing namespace ```myproject``` i just prefix the sting ```project-```.

Reply:
This looks to me like creating a confusing workaround for something that should be fixed elsewhere.

Reply:
The issue comes from the ginkgo json matcher - https://github.com/openshift/odo/pull/1612/files#diff-008acb9d2aae13e322536008a066d457R44. It seems the inbuilt matcher does not handle reordering.